migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java

```diff
@@ -206,109 +206,103 @@ private boolean migrationJobsExist() {
 
     @Override
     public void run() {
-        LOGGER.debug("Starting consistency checker controller for {} with period {}", migrationConfiguration.getConfigurationName(),
-                migrationConfiguration.getPeriod());
-        boolean interrupted = false;
-        long period = parsePeriod(migrationConfiguration.getPeriod());
-        while (!interrupted) {
-            interrupted = isInterrupted();
-            if (!interrupted) {
-                Breakpoint.checkpoint("CCC:start");
-                LOGGER.debug("Consistency checker {} woke up", migrationConfiguration.getConfigurationName());
-                // Lets update our configuration first
-
-                MigrationConfiguration newCfg = migrationConfiguration;
-                try {
-                    newCfg = reloadMigrationConfiguration();
-                } catch (Exception e) {
-                    LOGGER.error("Cannot load configuration", e);
-                }
-                if (newCfg == null) {
-                    interrupted = true;
-                    LOGGER.debug("Consistency checker {} configuration is no longer available", migrationConfiguration.getConfigurationName());
-                } else {
-                    migrationConfiguration = newCfg;
-                    // Lets recalculate period, just in case it changed
-                    period = parsePeriod(migrationConfiguration.getPeriod());
-
-                    if (migrationJobsExist()) {
-                        LOGGER.info("There are migration jobs for {}, not running consistency checker this time",
+        LOGGER.debug("Starting consistency checker controller for {} with period {}",migrationConfiguration.getConfigurationName(),
+                     migrationConfiguration.getPeriod());
+        long period=parsePeriod(migrationConfiguration.getPeriod());
+        while(!stopped) {
+            Breakpoint.checkpoint("CCC:start");
+            LOGGER.debug("Consistency checker {} woke up",migrationConfiguration.getConfigurationName());
+            // Lets update our configuration first
+            
+            MigrationConfiguration newCfg=migrationConfiguration;
```

Comment:
It runs once every period, and reloads the configuration to see if it should still run.